azurerm/resource_arm_dns_ns_record.go

```diff
@@ -33,9 +35,21 @@ func resourceArmDnsNsRecord() *schema.Resource {
 				Required: true,
 			},
 
+			"records": {
+				Type: schema.TypeList,
+				//TODO: add `MinItems: 1`` and `Required: true` once we remove the `record` attribute
```

Comment:
_minor_ `MinItems: 1` is implied by the fact that this would be required, so we only actually need `Required: true` :)

Reply:
Ah, good to know. I updated the TODO